### PR TITLE
Week 1 Homework

### PR DESCRIPTION
**Pull request by stateanomaly**

### CODE REVIEW

File: Layman-eUTxO-Design-Course/batch-1/stateanomaly/ergo-school-summary.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Ergo School
```
Comment (kii-dot) [inline Layman-eUTxO-Design-Course/batch-1/stateanomaly/ergo-school-summary.md]: great. Can you add it into batch-1/teams/ergo-school/ergo-school-summary.md?

There should be a teams folder, or team.

File: Layman-eUTxO-Design-Course/batch-1/stateanomaly/week1-homework.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# DeCo Laymans - Week 1 Homework
```
Comment (kii-dot) [inline Layman-eUTxO-Design-Course/batch-1/stateanomaly/week1-homework.md]: Can you add this into batch-1/individual/stateanomaly/week1-homework.md?

Comment (stateanomaly) [reply Layman-eUTxO-Design-Course/batch-1/stateanomaly/week1-homework.md]: Done!

File: Layman-eUTxO-Design-Course/batch-1/stateanomaly/ergo-school-summary.md (hunk 1 above)
Comment (stateanomaly) [reply Layman-eUTxO-Design-Course/batch-1/stateanomaly/ergo-school-summary.md]: Done!

File: Layman-eUTxO-Design-Course/batch-1/stateanomaly/ergo-school-summary.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Ergo School’s mission is to provide a learning platform for on-boarding Ergo developers at scale. We will do this through funneling the DeCo curriculum into a self-paced online learning course in which students build real-world projects. 
+Students will learn through doing and may pick and choose which projects to build. Each project will be designed to focus on utilizing one or a set of commonly used core concepts in building dapps on Ergo. Information will be funneled in from the DeCo curriculum and be layered and presented in an “as needed” basis to keep it fun and reduce information overload. The goal is to take beginners to a level where they can have a foundation to then self-study and build on their own. Community-building and creating an environment where community helps community is essential. This can evolve to be a parallel track to the DeCo live curriculum.
```
Comment (kii-dot) [inline Layman-eUTxO-Design-Course/batch-1/stateanomaly/ergo-school-summary.md]: as said before. EXCITED

File: Layman-eUTxO-Design-Course/batch-1/stateanomaly/ergo-school-summary.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Ergo School
+
+Ergo School’s mission is to provide a learning platform for on-boarding Ergo developers at scale. We will do this through funneling the DeCo curriculum into a self-paced online learning course in which students build real-world projects. 
+Students will learn through doing and may pick and choose which projects to build. Each project will be designed to focus on utilizing one or a set of commonly used core concepts in building dapps on Ergo. Information will be funneled in from the DeCo curriculum and be layered and presented in an “as needed” basis to keep it fun and reduce information overload. The goal is to take beginners to a level where they can have a foundation to then self-study and build on their own. Community-building and creating an environment where community helps community is essential. This can evolve to be a parallel track to the DeCo live curriculum. 
+
+## Example Project Structure: 
+1. Overview of project with demo of what we’re going to build.
+2. Text-based overview of fundamental eUTXO and Ergoscript concepts which will be used in the project. 
+3. Tutorial on setting up development environment. 
+4. Walk through of building the dapp.
```
Comment (kii-dot) [inline Layman-eUTxO-Design-Course/batch-1/stateanomaly/ergo-school-summary.md]: would you be having this dapp build for your teams first prototype?

**Comment by kii-dot** [comment]
I may have been confusing. Here's a few comments. And it should be good

File: Layman-eUTxO-Design-Course/batch-1/teams/ergo-school/ergo-school-summary.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Ergo School
```
Comment (kii-dot) [inline Layman-eUTxO-Design-Course/batch-1/teams/ergo-school/ergo-school-summary.md]: This file and folder can be deleted. Only keep the one in team. (I may have said teams in the past, my bad)

**Comment by kii-dot** [comment]
Ergoscript homework looks great. I enjoy the details of red box being its previous state.

**Comment by kii-dot** [comment]
can you ping us in discord when you made your edits?

**Pull request merged**